components/brave_wallet_ui/common/api/getKeyringsByType.ts

```diff
@@ -0,0 +1,52 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { assert } from 'chrome://resources/js/assert.m.js'
+import { TREZOR_HARDWARE_VENDOR, LEDGER_HARDWARE_VENDOR, KeyringControllerRemote } from 'gen/brave/components/brave_wallet/common/brave_wallet.mojom.m.js'
+import LedgerBridgeKeyring from '../../common/hardware/ledgerjs/eth_ledger_bridge_keyring'
+import TrezorBridgeKeyring from '../../common/hardware/trezor/trezor_bridge_keyring'
+export type HardwareKeyring = LedgerBridgeKeyring | TrezorBridgeKeyring
+
+const VendorTypes = [TREZOR_HARDWARE_VENDOR, LEDGER_HARDWARE_VENDOR] as const
+export type HardwareVendor = typeof VendorTypes[number]
```

Comment:
isn't this just `string`? Would it be more precice to do `type HardwareVendor = TREZOR_HARDWARE_VENDOR | LEDGER_HARDWARE_VENDOR`?